vd.js

```diff
@@ -0,0 +1,125 @@
+// || ========= Appending Speakers ========== ||
+
+const arrCards = [
```

Comment:
Your code iterates through the speakers list and creates HTML elements with the data from each object 🔥 Great work!